control/trajectory_follower/src/mpc_lateral_controller.cpp

```diff
@@ -65,6 +65,7 @@ MpcLateralController::MpcLateralController(rclcpp::Node & node) : node_{&node}
   m_mpc.m_admissible_yaw_error_rad = node_->declare_parameter<double>("admissible_yaw_error_rad");
   m_mpc.m_use_steer_prediction = node_->declare_parameter<bool>("use_steer_prediction");
   m_mpc.m_param.steer_tau = node_->declare_parameter<double>("vehicle_model_steer_tau");
+  m_enable_traj_extending = node_->declare_parameter<bool>("enable_traj_extending");
```

Comment:
(naming) the name simply as `enable_traj_extending` is not clear about its purpose. My suggestion is `extend_trajectory_in_yaw_direction`, `extend_trajectory_for_end_yaw_control` or something like developpers can understand its purpose.

Reply:
I chose `extend_trajectory_for_end_yaw_control`
fixed in https://github.com/autowarefoundation/autoware.universe/pull/2447/commits/c0cf295e6c81d5fd4382415f3096a18da30f9ac5